testsuite/integration/manualmode/pom.xml

```diff
@@ -150,7 +161,6 @@
                     <plugin>
                         <groupId>org.codehaus.mojo</groupId>
                         <artifactId>exec-maven-plugin</artifactId>
-                        <version>1.6.0</version>
```

Comment:
Is there any reason for removing this line?

Reply:
already defined

Reply:
where exactly?

Reply:
jboss-parent-35.pom line388